src/components/src/BulkOrder/bulkorder.main.tsx

```diff
@@ -296,15 +451,19 @@ class BulkOrder extends Component<BulkOrderProps, BulkOrderState> {
           <div className="tab-content">
             <div className="tab-pane fade show active" id="quick-order" role="tabpanel" aria-labelledby="quick-order-tab">
               <div className="form-content form-content-submit col-sm-offset-4">
-                <button
-                  className="ep-btn primary small btn-itemdetail-addtocart"
-                  id="add_to_cart_quick_order_button"
-                  disabled={isDisabledButton}
-                  type="submit"
-                  onClick={() => { this.addAllToCart(items, true); }}
-                >
-                  {intl.get('add-all-to-cart')}
-                </button>
+                {multiCartData && multiCartData._carts ? (
+                  <SelectCartButton isQuickOrder />
+                ) : (
+                  <button
+                    className="ep-btn primary small btn-itemdetail-addtocart"
+                    id="add_to_cart_quick_order_button"
+                    disabled={quickOrderDisabledButton}
+                    type="submit"
+                    onClick={() => { this.addAllToCart(items, true); }}
```

Comment:
Minor comment. The "item added to cart" popup will not get triggered by calling addAllToCart. If it's easy to, we should have it display here as well for both multi-cart and single cart